api/src/main/java/com/example/api/user/service/DashboardService.java

```diff
@@ -161,10 +179,44 @@ private Long getSurveysNumber(CourseMember member) {
         return surveyResultRepository.countAllByMember(member);
     }
 
+    private List<Auction> getResolvedAuctions() {
+        return auctionRepository.findAllResolved();
+    }
+
+    private Double getStudentAuctionsWonCount(CourseMember member) {
+        double count = 0;
+        List<Auction> resolvedAuctions = getResolvedAuctions();
+
+        if(!resolvedAuctions.isEmpty()) {
```

Comment:
raczej nie trzeba tego sprawdzać, jak będzie pusta to for się po prostu nie wykona

Reply:
wywala sie kafelek jak nie sprawdzimy